### ISSUE TITLE
--append replaces the first few HTML files with new ones when --split is used

### ISSUE THREAD

**Issue by guicaz** [bepaald/signalbackup-tools#193]
Today I tried to take a newer backup of my signal chats and wanted to append what I already exported to today's date. After the export was successful, the index.html file was properly updated, but when I looked into the folder for one conversation the number of files didn't change. I noticed after a bit that it overwrote the first few html files with new ones. I haven't tested without the `--split` option if it appends properly. Is there an option I missed or is this a bug? What I expected to happen is that it reads what is the last .html number and continues from that point on.

This is the command I used:

    ./signalbackup-tools ../signal-2024-03-08-00-22-45.backup -p <passphrase> --exporthtml ../SignalExportHTML/ --limittodates 1693458000000,1704063599000 --split --append

**Comment by hugogithubs** [comment]
Hello,
Quick side note / remark: 
I also noticed in the past, that, if user IDs changed, it is not possible to append correctly and new threads with new IDs where created in addition to the old (old ones will still be retained, but will of course no longer be linked in the index file). Since the program has no knowledge aboout changed IDs, my issue is probably not solveable.

**Comment by guicaz** [comment]
@bepaald 

You are right, I did not understand exactly how these options worked. Your message explains a lot. So if I get this right, `--append` only skips exporting medias that already exists, and *appends* new ones, right? I don’t know of a better name, maybe change or add to the description that it "writes newer medias without overwriting previous ones". But it is probably my fault for not understanding right.

I also indeed didn’t understand how `--split` worked, but now I do, thank you for the explaination.

Thanks for the quick reply, and software.
